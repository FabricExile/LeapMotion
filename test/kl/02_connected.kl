require LeapMotion;

operator entry() {
  LeapController controller();
  report(controller.isConnected());
  report('completed!'); 
}