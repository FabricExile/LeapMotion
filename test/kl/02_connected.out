[FABRIC:MT] false
[FABRIC:MT] completed!
